src/WebJobs.Script/Description/FunctionDescriptorProvider.cs

```diff
@@ -72,6 +73,21 @@ public virtual bool TryCreate(FunctionMetadata functionMetadata, out FunctionDes
             }
         }
 
+        public void VerifyResolvedBindings(FunctionMetadata functionMetadata, IEnumerable<FunctionBinding> inputBindings, IEnumerable<FunctionBinding> outputBindings)
+        {
+            IEnumerable<string> bindingsFromMetadata = functionMetadata.InputBindings.Select(f => f.Type).Union(functionMetadata.OutputBindings.Select(f => f.Type));
+            IEnumerable<string> resolvedBindings = inputBindings.Union(outputBindings).Select(b => b.Metadata.Type);
+            IEnumerable<string> unresolvedBindings = bindingsFromMetadata.Where(b => !resolvedBindings.Contains(b));
+
+            string allUnresolvedBindings = string.Join(", ", unresolvedBindings);
```

Comment:
nit: change the `IsNullOrEmpty` check below to `unresolvedBindings.Any()`, and move this line into the conditional block so you only have to do this if there are unresolved bindings.